res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -326,60 +327,6 @@ limitations under the License.
     }
 }
 
-.mx_AppPermissionWarning_row {
-    margin-bottom: 12px;
-}
-
-.mx_AppPermissionWarning_smallText {
-    font-size: $font-12px;
-}
-
-.mx_AppPermissionWarning_bolder {
-    font-weight: var(--font-semi-bold);
-}
-
-.mx_AppPermissionWarning_helpIcon {
-    margin-top: 1px;
-    margin-right: 2px;
-    width: 10px;
-    height: 10px;
-    display: inline-block;
-
-    &::before {
-        display: inline-block;
-        background-color: $accent;
-        mask-repeat: no-repeat;
-        mask-size: 12px;
-        width: 12px;
-        height: 12px;
-        mask-position: center;
-        content: "";
-        vertical-align: middle;
-        mask-image: url("$(res)/img/feather-customised/help-circle.svg");
-    }
-}
-
-.mx_AppPermissionWarning_tooltip {
```

Comment:
I noticed that this class name had not followed our naming policy, since this class has been applied to tooltip, which is outside of `mx_AppPermissionWarning`. Therefore, [this line above](https://github.com/matrix-org/matrix-react-sdk/pull/10824/files#diff-a9bc8899b6e217c49aaa8a7ec1a54862b6f6a39e09d5e9d4b5f54e8958843e4eR57) causes a regression of styling a tooltip on the widget permission approval UI, which appears by hovering on the question mark.

Going to address the issue.